app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -217,6 +222,26 @@ private void renderSessions(List<SessionViewModel> adjustedSessionViewModels) {
             binding.txtDate.setText(adjustedSessionViewModels.get(0).getFormattedDate());
             binding.txtDate.setVisibility(View.VISIBLE);
         }
+        scrollTo(extractTodayPos(getContext(), new Date(), adjustedSessionViewModels));
+    }
+
+    private int extractTodayPos(Context context, Date today, List<SessionViewModel> adjustedSessionViewModels) {
+        String todayString = DateUtil.getMonthDate(today, context);
+        return Stream.of(adjustedSessionViewModels)
+                .map(SessionViewModel::getFormattedDate)
+                .indexed()
+                .filter(pair -> todayString.equals(pair.getSecond()))
+                .findFirst().orElse(new IntPair<>(0, "")).getFirst();
+    }
+
+    private void scrollTo(int position) {
+        binding.recyclerView.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+            @Override
+            public void onGlobalLayout() {
+                binding.recyclerView.getLayoutManager().scrollToPosition(position);
+                binding.recyclerView.getViewTreeObserver().removeOnGlobalLayoutListener(this);
```

Comment:
Sorry, I don't know why `OnGlobalLayoutListener` is needed 🙇 
Is it enough just to call `binding.recyclerView.getLayoutManager().scrollToPosition(position);` ?